tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/upgrade/action/ImportProjectNextActionHandler.java

```diff
@@ -76,6 +79,8 @@ protected Object run( Presentation context )
                 ILiferayProjectImporter importer = LiferayCore.getImporter( "gradle" );
 
                 importer.importProject( wsPath, npm );
+
+                importSDKProject( new Path( wsPath ).append( "plugins-sdk" ), npm );
```

Comment:
Hey Greg, in here it is not work now, as when we import lr-ws(with plugins-sdk) we don't change any build.xxx.properties by "blade init" commond, for commond line it ok but in ide side we need to read build.xxx.properties and get runtime path and some else. I will redesign and talk about it with you in next commit.